### PR TITLE
Cromwell 28.2

### PR DESCRIPTION
**Pull request by ruchim**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by ruchim** [comment]
Thanks @ilovezfs for getting the build to pass! Can you please explain your changes?

File: Formula/cromwell.rb [hunk 1]
```diff
@@ -1,9 +1,8 @@
 class Cromwell < Formula
   desc "Workflow Execution Engine using Workflow Description Language"
   homepage "https://github.com/broadinstitute/cromwell"
-  url "https://github.com/broadinstitute/cromwell/releases/download/28/cromwell-28.jar"
-  version "28.1"
-  sha256 "c9ce762df236588ded042ceaf099848c0c1685d34788442ee251615ff13b5190"
+  url "https://github.com/broadinstitute/cromwell/releases/download/28.2/cromwell-28_2.jar"
```
Comment (fxcoudert) [inline Formula/cromwell.rb]: Is the version correctly autodetected as `28.2`, or does it think it's `28_2`?

Comment (ilovezfs) [inline Formula/cromwell.rb]:
```
Josephs-MacBook-Pro:~ joe$ brew irb
==> Interactive Homebrew Shell
Example commands available with: brew irb --examples
irb(main):001:0> Version.detect("https://github.com/broadinstitute/cromwell/releases/download/28.2/cromwell-28_2.jar", {})
=> #<Version::FromURL:0x007f8bf3a4e6e8 @version="28.2">
irb(main):002:0>
```

**Comment by ilovezfs** [comment]
Thanks for your first contribution to Homebrew, @ruchim! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request merged**